Gordon360/Models/ViewModels/StudentProfileViewModel.cs

```diff
@@ -123,6 +125,8 @@ public record StudentProfileViewModel
                 stu.Gender ?? "",
                 stu.grad_student ?? "",
                 stu.GradDate ?? "",
+                stu.PlannedGradYear ?? "",
+                stu.IsPlannedGradYearPrivate == 1 ? true : false,
```

Comment:
IsPlannedGradYearPrivate is stored as a `bool`. SQL Stores booleans as a bit (which is logical T/F). You do not need to cast to an int and back.

Reply:
It is the same as IsMobilePhonePrivate 
![image](https://github.com/gordon-cs/gordon-360-api/assets/115328139/66fff51d-95cc-429f-89c3-6a0f9e025714)
It is stored as an int
![image](https://github.com/gordon-cs/gordon-360-api/assets/115328139/7882b0cf-9d62-460f-ac6e-89ebf50a47fd)

Reply:
as below

Reply:
Right, which is why it should not be stored as an `int`. It should be stored as a `bit`